### ISSUE TITLE
Syntax error in "INSERT INTO" mixpanel events

### ISSUE THREAD

**Issue by markoengelman** [mixpanel/mixpanel-swift#564]
We are experiencing an issue since upgrading to version 4.0.X. (logs are from latest version 4.0.3)

Same as in this #556 , the tracking events are not showing up in Mixpanel Dashboard, and we are also getting an syntax error in the debug log from MixPanel. 

**Log**

```
2022-10-26 10:20:05.423072+0200 Mediately [34212:311314] [logging] near ".": syntax error in "INSERT INTO mixpanel_com.modrajagoda.mixpanel_events (data, flag, time) VALUES(?, ?, ?);"
2022-10-26 10:20:05.423915+0200 Mediately [34212:311314] [logging] near ".": syntax error in "CREATE TABLE IF NOT EXISTS mixpanel_com.modrajagoda.mixpanel_events(id integer primary key autoincrement,data blob,time real,flag integer);"
```

**Comment by jaredmixpanel** [comment]
@markoengelman similar to #556 you're getting this error because of the `.` in your `instanceName` "com.modrajagoda.mixpanel", if you remove or replace the `.` in that string, for example "modrajagod" or "com_modrajagoda_mixpanel" this error should go away. However, in the next update I'll make sure we handle this so that special characters in `instanceName`'s don't cause this problem. Thanks for bringing it to our attention. I'll leave this issue open until that update is released.

**Comment by jaredmixpanel** [comment]
@markoengelman this is fixed in v4.0.4
